skrf/vi/vna/hp8510c_sweep_plan.py

```diff
@@ -0,0 +1,186 @@
+import dataclasses
+import numpy as np
+from typing import List,Union
+
+class SweepSection:
+    def get_hz(self):
+        raise NotImplementedError()
+    def apply_8510(self, hp8510c):
+        raise NotImplementedError()
+    def mask_8510(self, network):
+        return network
```

Comment:
If the method `SweepSection` should not be instantiated directly but only be inherited, I guess it should be defined as an abstract class, no? Something like that?

```suggestion
from typing import List,Union
from abc import ABC, abstractmethod

class SweepSection(ABC):
    @abstractmethod
    def get_hz(self):
        pass
    @abstractmethod
    def apply_8510(self, hp8510c):
        pass
    def mask_8510(self, network):
        return network
```